src/main/scala/edu/cmu/cs/obsidian/codegen/CodeGen.scala

```diff
@@ -380,9 +380,9 @@ class CodeGen (val target: Target, table: SymbolTable) {
         }
 
         val meth: JMethod = newClass.method(JMod.PUBLIC, javaRetType, transaction.name)
+        addTransactionExceptions(meth, translationContext)
         meth._throws(model.directClass("edu.cmu.cs.obsidian.client.ChaincodeClientAbortTransactionException"))
```

Comment:
Can we remove this _throws line now?

Reply:
Not really, though of course we could move it somewhere, but I'm not sure what would be better; `addTransactionExceptions` only adds the `ChaincodeClientAbortTransactionException` to the throws list in the case that the target is a Client rather than a Server, but that's not the case here.